docs/internal/epic-delivery-workflow.md

```diff
@@ -0,0 +1,89 @@
+# Cross-Team Epic Delivery Workflow
+
+This guide documents the optimal steps to create and work on Feature Epics in a collaborative way.
+
+## Table of Contents
+
+- [Cross-Team Epic Delivery Workflow](#cross-team-epic-delivery-workflow)
+  - [Table of Contents](#table-of-contents)
+  - [1. Epic Creation (Product / Dev / Design)](#1-epic-creation-product--dev--design)
+  - [2. Design Exploration Phase](#2-design-exploration-phase)
+  - [3. Final Design Approval \& Handoff](#3-final-design-approval--handoff)
+  - [4. Development Planning](#4-development-planning)
+  - [5. Design QA (Post-Implementation Review)](#5-design-qa-post-implementation-review)
+  - [6. Epic Completion](#6-epic-completion)
+  - [TLDR Summary Workflow (High-Level)](#tldr-summary-workflow-high-level)
+
+## 1. Epic Creation (Product / Dev / Design)
+
+* An **EPIC** is created for any new feature or major improvement (by product, dev or design teams).
+* The EPIC includes:
+    * Feature requirements
+    * High-level scope
+    * Known constraints or assumptions
+* The epic is assigned to the **Design Team** with the label **<code>needs-design</code>**.
+
+
+## 2. Design Exploration Phase
+
+* The designer changes the epic status to **“In Progress”** once exploration begins.
+* The designer starts drafting early concepts / prototypes.
+* The designer is responsible for **pinging the Dev owner** early to initiate collaboration.
+* Multiple review iterations may happen with: 
+    * Dev / Product / (Optionally) other stakeholders (eg QA engineers)
+* Syncs may be:
+    * **Async** (chat / comments) / **Sync calls**, as needed
+* Communication should be **frequent**, potentially more than once per week for complex features.
+
+
+## 3. Final Design Approval & Handoff
+
+* Last meeting to present the final design
+* Once iterated and aligned, the final design direction is agreed by:
+    * Design / Product / Dev
+* The designer prepares and delivers the **Figma handoff package**.
+* The designer **removes the <code>needs-design</code> label** from the epic.
+* The designer **pings the Dev team** (in the same epic) to indicate the feature is ready for implementation.
+
+
+## 4. Development Planning
+
+* The Dev owner reviews the final design.
+* The Dev owner breaks down the epic into **small, actionable subtasks**.
+* Subtasks include:
+    * UI components / Interactions / Edge states
+    * Integration work
+    * Non-visual logic affected by UI
+* The Dev owner assigns subtasks within the team (if needed) and starts implementation.
+* It should have acceptance criteria
+
+
+## 5. Design QA (Post-Implementation Review)
+
+* Once the implementation is complete, **Dev notifies Design**.
+* A **Design QA task** is created within the epic for tracking.
```

Comment:
Maybe this could be included in point 4, so the necessary subtasks to close the epic are already created from the very beginning. WDYT?